src/NuGet.Clients/NuGet.CommandLine/Commands/PackCommand.cs

```diff
@@ -75,6 +75,9 @@ public ICollection<string> Exclude
         [Option(typeof(NuGetCommand), "CommandMSBuildVersion")]
         public string MSBuildVersion { get; set; }
 
+        [Option(typeof(NuGetCommand), "CommandMSBuildPath")]
+        public string MSBuildPath { get; set; }
```

Comment:
We need a couple tests for this in NuGet.CommandLine.Test:

1. Verify that `MSBuildPath` is used.
1. Verify that `MSBuildPath` is preferred over `MSBuildVersion`.
1. Verify what happens when `MSBuildPath` is pointing to a file that does not exist.